packages/@okta/vuepress-site/docs/reference/api/rulesets/index.md

```diff
@@ -0,0 +1,979 @@
+-H "Authorization: SSWS ${api_token}" \
+"https://${yourOktaDomain}/api/v1/policies/{rulesetId}"
+```
+
+##### Response Types
+
+HTTP 204:
+*No Content*
+
+### Update a Rule Set
+
+<ApiOperation method="put" url="/api/v1/policies/${rulesetId}" />
+
+##### Request Parameters
+
+The rule set ID described in the [RuleSet object](#rule-set-object) is required.
+
+##### Request Example
+
+```bash
+curl -v -X PUT \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+-d '{
+  "type": "Okta:SignOn",
+  "id": "rst30qv3hcOFmC0GL0g7",
+  "status": "ACTIVE",
+  "name": "Default Policy",
+  "default": true
+  }
+}' "https://${yourOktaDomain}/api/v1/policies/{rulesetId}"
+```
+
+##### Response Types
+
+HTTP 200:
+[RuleSet object](#rule-set-object)
+
+### Create a Rule Set
+
+<ApiOperation method="post" url="/api/v1/policies" />
+
+##### Request Example
+
+```bash
+curl -v -X POST \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+-d '{
+  "type": "Okta:SignOn",
+  "status": "ACTIVE",
+  "name": "My New Ruleset",
+  "default": false
+  }
+}' "https://${yourOktaDomain}/api/v1/policies"
+```
+
+##### Response Types
+
+HTTP 204:
+[RuleSet object](#rule-set-object)
+
+### Activate a Rule Set
+
+<ApiOperation method="post" url="/api/v1/policies/${rulesetId}/lifecycle/activate" />
+
+##### Request Parameters
+
+The rule set id described in the [RuleSet object](#rule-set-object) is required.
+
+##### Request Example
+
+```bash
+curl -v -X POST \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://${yourOktaDomain}/api/v1/policies/{rulesetId}/lifecycle/activate"
+```
+
+##### Response Types
+
+HTTP 204:
+*No Content is returned when the activation is successful.*
+
+### Deactivate a Rule Set
+
+<ApiOperation method="post" url="/api/v1/policies/${rulesetId}/lifecycle/deactivate" />
+
+##### Request Parameters
+
+The rule set ID described in the [RuleSet object](#rule-set-object) is required.
+
+##### Request Example
+
+```bash
+curl -v -X POST \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://${yourOktaDomain}/api/v1/policies/{rulesetId}/lifecycle/deactivate"
+```
+
+##### Response Types
+
+HTTP 200:
+*No Content is returned when the deactivation is successful.*
+
+## Rule Operations
+
+### Get Rules in a Rule Set
+
+<ApiOperation method="get" url="/api/v1/policies/${rulesetId}/rules" />
+
+##### Request Parameters
+
+The rule set ID described in the [RuleSet object](#rule-set-object) is required.
+
+##### Request Example
+
+```bash
+curl -v -X GET \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://${yourOktaDomain}/api/v1/policies/{rulesetId}/rules"
+```
+
+##### Response Types
+
+HTTP 200:
+[Rule object](#rule-object)
```

Comment:
This would be an Array of Rules, right?

Reply:
Correct, if you are requesting `https://${yourOktaDomain}/api/v1/policies/{rulesetId}/rules` you will get a list of rules.
`https://${yourOktaDomain}/api/v1/policies/{rulesetId}/rules/{ruleId}` will be a single rule